roles/matrix-etherpad/tasks/validate_config.yml

```diff
@@ -0,0 +1,7 @@
+- name: Fail if required Etherpad settings not defined
+  fail:
+    msg: >-
+      You need to define a required configuration setting (`{{ item }}`) for using Etherpad.
+  with_items:
+    -
```

Comment:
Anything to add here?

Reply:
Good catch! That came by copypasting Dimension tasks.
That would be the right place to verify that Dimension is installed.